src/libs/actions/IOU.js

```diff
@@ -49,34 +49,48 @@ Onyx.connect({
  * @param {Object} report
  * @param {Number} amount - always in the smallest unit of the currency
  * @param {String} currency
- * @param {String} recipientEmail
+ * @param {String} payeeEmail
  * @param {Object} participant
  * @param {String} comment
  */
-function requestMoney(report, amount, currency, recipientEmail, participant, comment) {
-    const debtorEmail = OptionsListUtils.addSMSDomainIfPhoneNumber(participant.login);
+function requestMoney(report, amount, currency, payeeEmail, participant, comment) {
+    const payerEmail = OptionsListUtils.addSMSDomainIfPhoneNumber(participant.login);
     let chatReport = lodashGet(report, 'reportID', null) ? report : null;
+    const isPolicyExpenseChat = participant.isPolicyExpenseChat || participant.isOwnPolicyExpenseChat;
     let isNewChat = false;
+
+    // If this is a policyExpenseChat, the chatReport must exist and we can get it from Onyx
+    if (!chatReport && isPolicyExpenseChat) {
+        chatReport = chatReports[`${ONYXKEYS.COLLECTION.REPORT}${participant.reportID}`];
```

Comment:
This is super confusing, why is it participant?
```suggestion
        chatReport = chatReports[`${ONYXKEYS.COLLECTION.REPORT}${report.reportID}`];
```

Reply:
Because the report is null if we initiate the flow from the global create. However, the participant always stores the reportID if it exists.

Reply:
I added a comment.